geth/node/status_node.go

```diff
@@ -98,13 +99,34 @@ func (n *StatusNode) Start(config *params.NodeConfig, services ...node.ServiceCo
 		return err
 	}
 
+	statusDB, err := db.Create(filepath.Join(n.config.DataDir, params.StatusDatabase))
```

Comment:
Do we know:
- how much data does this database keep in the memory?
- overall, how much data do we expect to store here?
- how often does it dump it to the disk? I/O ops/s?

Reply:
It's not me who introduced that DB 😀
Regarding the deduplication cache, we expect to store 2 days worth of data there. As soon as the day change, we clean-up older entries.
It's about 2MB disk space per 10000 unique messages.

Reply:
Yeah, the previous usage was kind of research but it's true that it should be asked when it was introduced. But this should be investigated anyway. `leveldb` has a lot of configuration options.

Reply:
I ran some benchmarks (you can see on in tests) and performance of that configuration seems pretty good. 20ms on average to deduplicate 300 messages while keeping ~60000 in the cache.

Reply:
in my defense I can say that previous usage was super low, like i could store there maybe 20 enodes in total, each one less than 100 bytes

but in general, even current usage should be nothing in comparison to les

btw what storage react uses for persistent data?

Reply:
Not a big deal, we just need to be more careful when we add this kind of stuff. And I was also curious how efficient it is.

It's called Realm but I don't know which versions and project exactly. It should be in status-react dependencies.

Reply:
can't find benchmarks for realm vs leveldb. but i found out that leveldb supports less features, so it is probably more performant